worldedit-bukkit/src/main/java/com/sk89q/bukkit/util/ClassSourceValidator.java

```diff
@@ -44,6 +45,19 @@
 
     private static final Logger LOGGER = LogManagerCompat.getLogger();
     private static final String SEPARATOR_LINE = Strings.repeat("*", 46);
+    private static final Method loadClass;
+
+    static {
+        Method tmp;
+        try {
+            tmp = PluginClassLoader.class.getDeclaredMethod("loadClass0",
+                    String.class, boolean.class, boolean.class, boolean.class);
+            tmp.setAccessible(true);
+        } catch (NoSuchMethodException e) {
+            tmp = null;
+        }
```

Comment:
Should we log here? Not sure what severity / exact text, but seems like a reasonable thing to note.